includes/blocks/safe-svg/register.php

```diff
@@ -67,7 +67,7 @@ function render_block_callback( $attributes ) {
 	return apply_filters(
 		'safe_svg_inline_markup',
 		sprintf(
-			'<div class="safe-svg-cover" style="text-align: %s;">
+			'<div class="wp-block-safe-svg-svg-icon" style="text-align: %s;">
```

Comment:
Same suggestion as above
```suggestion
			'<div class="wp-block-safe-svg-svg-icon safe-svg-cover" style="text-align: %s;">
```

Reply:
I have updated this.